### DISCUSSION TITLE
Does this work for Galaxy A25?

### DISCUSSION THREAD

**Discussion by kartoofy** [tribalfs/GalaxyMaxHzPub#242] [General]
Does this work for Galaxy A25?

**Comment by tribalfs** [comment]
Not sure if refresh rate features will work on this device. We haven't tested on this device. But you may try it to find out.

**Reply by Ecephi** [reply]
I used it on my galaxy a25 and it works fine

**Reply by Ecephi** [reply]
Also btw, i wanna buy the premium, how much is it? I cant find the price

**Comment by 0x11DFE** [comment]


**Reply by 0x11DFE** [reply]
Actually it still works, the issue i had on my side is LiveBridge, It was constantly making the screen set to 120Hz because of the non-stop status bar updates.
